js/components/tabIcon.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const ImmutableComponent = require('./immutableComponent')
+const {StyleSheet, css} = require('aphrodite')
+const globalStyles = require('./styles/global')
+
+class TabIcon extends ImmutableComponent {
+  render () {
+    const className = css(
+      styles.icon,
+      this.props.withBlueIcon && styles.blueIcon
+    )
+    return <div className={className}>
+      <span className={this.props.styles} />
+    </div>
+  }
+}
+
+class AudioTabIcon extends ImmutableComponent {
+  render () {
+    return <TabIcon withBlueIcon {...this.props} />
```

Comment:
I wonder if there's a clean way to eliminate the mapping, since it's almost redundant;
ex: this returns withBlueIcon, which then maps itself to styles.blueIcon...

Can we have an array of styles? and then expand using a spread (in the parent)
`<TabIcon customStyles={[styles.blueIcon]} {...this.props} />`